### PR TITLE
[Ayushi Yadav] IP

### PR DESCRIPTION
**Pull request by ayushi0803**

### CODE REVIEW

File: src/main/java/Duke.java [hunk 1]
```diff
@@ -1,10 +1,63 @@
...
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("____________________________________________________________");
```
Comment (Sukkaito) [inline src/main/java/Duke.java:5]: I think you can just keep the indentation of these "separator line" (?), or extract them into a separate function and call the function instead.

File: src/main/java/Task.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
+public class Task {
+  protected String description;
+  protected boolean isDone;
+  protected String type;
```
Comment (Sukkaito) [inline src/main/java/Task.java:4]: Perhaps a better name for `type` could be more intuitive here?

File: src/main/java/Task.java
Comment (Sukkaito) [inline src/main/java/Task.java:1]: Should the indentation for this file be 4 spaces instead of 2?

File: src/main/java/Duke.java [hunk 3]
```diff
@@ -1,10 +1,63 @@
...
+            } else if (input.startsWith("event")) {
+                String taskDescription = input.substring(6);
+                tasks[taskCount] = new Task(taskDescription, "E");
+                taskCount++;
+                System.out.println("added: " + taskDescription);
+                System.out.println("____________________________________________________________");
+            }
+        }
+
+        scanner.close();
```
Comment (Sukkaito) [inline src/main/java/Duke.java:61]: Nice! You did remember to close the `scanner`! 👍 (Not like me)

**Review by Sukkaito — COMMENTED** [review]
In general, LGTM! 👍

File: src/main/java/Duke.java (hunk 1 above)
Comment (Yvorm) [reply src/main/java/Duke.java:5]: I think this is a neat idea - especially if you want to modularise your responses later down the line.

File: src/main/java/Duke.java [hunk 4]
```diff
@@ -1,10 +1,63 @@
...
+                System.out.println("added: " + taskDescription);
+                System.out.println("____________________________________________________________");
+            } else if (input.startsWith("event")) {
+                String taskDescription = input.substring(6);
+                tasks[taskCount] = new Task(taskDescription, "E");
+                taskCount++;
+                System.out.println("added: " + taskDescription);
+                System.out.println("____________________________________________________________");
+            }
+        }
```
Comment (Yvorm) [inline src/main/java/Duke.java:59]: I like the structure of this segment! I noticed some of the repeated could be abstracted into a function call. Were you planning on adding unique functionality to each case?

**Pull request closed without merging**